.github/workflows/diamond-storage-check.yml

```diff
@@ -0,0 +1,60 @@
+name: Check For Diamond Storage Changes
+
+on:
+  push:
+    branches:
+      - development
+    paths:
+      - '**.sol'
+  pull_request:
+    paths:
+      - '**.sol'
+
+jobs:
+  provide_contracts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - id: set-matrix
+        working-directory: packages/contracts
+        run: |
+          forge tree | grep -E '^src/dollar/libraries/Lib' | cut -d' ' -f1 | xargs basename -a | cut -d'.' -f1 | xargs -I{} echo src/dollar/libraries/{}.sol:{} >> contracts.txt
+          echo "::set-output name=matrix::$(cat contracts.txt | jq -R -s -c 'split("\n")[:-1]')"
```

Comment:
Pls [refactor](https://github.com/ubiquity/ubiquity-dollar/pull/795/files#diff-44b0878aa3a2c194318bbbbde7166259a120f3ad0a3497153359dd03db11f32fR27) to `GITHUB_OUTPUT`